build/kj_test.bzl

```diff
@@ -18,6 +23,20 @@ def kj_test(
         }),
         data = data,
         tags = tags,
-        size = size,
         **kwargs
     )
+    sh_test(
```

Comment:
This doubles the amount of tests we will run. Have we considered the impact on CI performance this will have?

Reply:
I believe this is inevitable, if we want to increase our test coverage. (PS: EW is following the same pattern as well)

Reply:
It is, but it also has a lot more than 4 cores available 😄
I don't want to be blocking on this change, just saying that this means we'll have to think harder about CI performance with this change.

Reply:
The workerd tests take a tiny tiny fraction of the time of the internal codebase tests, so it's probably fine.